x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_routing/review_suggestions_form/suggested_stream_panel.tsx

```diff
@@ -44,28 +48,22 @@ export function SuggestedStreamPanel({
   onPreview(toggle: boolean): void;
   onSuccess(): void;
 }) {
-  const [isModalOpen, toggleModal] = useToggle(false);
   const { timeState } = useTimefilter();
   const matchRate = useMatchRate(definition, partition, timeState.start, timeState.end);
   const selectedPreview = useStreamSamplesSelector((snapshot) => snapshot.context.selectedPreview);
+  const suggestedRuleId = useStreamsRoutingSelector((snapshot) => snapshot.context.suggestedRuleId);
+  const isUnderReview = suggestedRuleId === partition.name;
```

Comment:
This should also ensure we are in the right state to render the modal:
```tsx
const isUnderReview = useStreamsRoutingSelector(
    (snapshot) =>
      snapshot.matches({ ready: 'reviewSuggestedRule' }) &&
      snapshot.context.suggestedRuleId === partition.name
  );
```

Reply:
Fixed in https://github.com/elastic/kibana/pull/236758/commits/c8457a88f9d7d3014c91a4bbf2c78dcdbc12c642